src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper.cc

```diff
@@ -8,11 +8,11 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * Unless required by applicable law or agreed to in writing, software *
```

Comment:
Looks like the formatting got a bit messed up here.

Reply:
Fixed.